prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -0,0 +1,136 @@
+import pandas as pd
+from powersimdata.utility.distance import haversine
+
+from prereise.gather.griddata.hifld import const
+from prereise.gather.griddata.hifld.data_access import load
+
+
+def floatify(value, default=pd.NA):
+    """Return a float if possible, otherwise return a default value.
+
+    :param object value: value to be coerced to float, if possible.
```

Comment:
object --> str (from what I can see in the `isinstance`)

Reply:
It could be any object type, even a float already. We just have extra logic to assist with strings.